slang-gfx.h

```diff
@@ -1424,6 +1424,13 @@ class IDevice: public ISlangUnknown
         DeviceType deviceType = DeviceType::Default;
         // Name to identify the adapter to use
         const char* adapter = nullptr;
+        // Device handles (if they already exist)
+        struct existingDeviceHandles
```

Comment:
We do have a convention that type names are BigCamel. So here should be ExistingDeviceHandle.

Reply:
Derp, fixed.